src/core-sdk-tasks/BuildFPMToolPreReqs.cs

```diff
@@ -183,14 +179,11 @@ private string BuildCmdParameters(ConfigJson configJson, string package_version)
 
             // Map all the payload directories as they need to install on the system 
             if (configJson.Install_Root != null)
```

Comment:
Time for me to rant. I think our general style guidelines have always been to have braces for `if` statements, though if you really want to you may be able to use conditionals and discard statements. Not a big issue since this code has been around for a long time I think

Reply:
Oh definitely. But there's code in the SDK repo that doesn't follow ANY guidelines... that was written in 2006, originally from .NET Framework. I think there are more formatting guidelines I'd like to turn on in the SDK repo, but I don't have a complete list yet. I'll mention I didn't do "complete" cleanup to the code here. It was more like spot checks and some automatic tasks in VS. I did a lot of these in December when I was trying to understand the scope and projects in this repo for migration.